container/core.py

```diff
@@ -431,7 +431,9 @@ def hostcmd_version(base_path, project_name, engine_name, **kwargs):
 
 
 @host_only
-def hostcmd_import(base_path, project_name, engine_name, **kwargs):
+def hostcmd_import(base_path, project_name, engine_name, force=False, **kwargs):
+    kwargs['force'] = force
```

Comment:
Since we're not actually using `force` here, we don't need to expose it in the parameters, and therefore don't need this line. Just accept that it's part of `kwargs`, and allow it to be passed along.